editor/README_CURSIVE.md

```diff
@@ -4,39 +4,43 @@
+Once the IntelliJ restarts, we should make some changes to the Cursive preferences to make it easier to work with. You are not required to change these preferences, but we recommend you do, especially if this is your first time using Clojure.
 
-1. Enable **Rainbow parentheses**. This colors parentheses and brackets in the editor so it will be easier to distinguish between nested scopes.
+On the Welcome Screen, select **Customize** from the menu on the right and click the **All settings...** link at the bottom. Inside the **Preferences** window that appears, navigate to the **Languages & Frameworks > Clojure** page.
 
-2. Disable **Highlight unresolved symbols**. For some reason Cursive is not able to resolve all the symbols in the editor project. When enabled, this clutters up the editor with a lot of noisy highlighting that doesn't really help.
+1. Enable **Rainbow parentheses**. This colors parentheses and brackets in the editor so it will be easier to distinguish between nested scopes.
 
-3. Disable **Load out-of-date file dependencies transitively**. By default, Cursive will automatically reload any edited files that are referenced from the file you're reloading. This can cause type errors if a referenced file re-defines a Java class. It is better to manually refresh only the changed forms, because once you trigger a type error the editor needs to be restarted to get it back on track.
+2. Disable **Load out-of-date file dependencies transitively**. By default, Cursive will automatically reload any edited files that are referenced from the file you're reloading. This can cause type errors if a referenced file re-defines a Java class. It is better to manually refresh only the changed forms, because once you trigger a type error the editor needs to be restarted to get it back on track.
 
-4. Disable **Add forms send from the editor to REPL history**. It is preferable to have the REPL history only contain the evaluations you sent from the REPL window when experimenting.
+3. Disable **Add forms sent from the editor to REPL history**. It is preferable to have the REPL history only contain the evaluations you sent from the REPL window when experimenting.
 
 ## Setting up the Defold Editor project
 
 Time to open the Defold Editor project. The first time you open it, you'll need to configure some things.
 
-1. Choose **Open** from the Welcome screen and select the `defold/editor` directory. Choose **Open as Project** when prompted.
+1. Choose **Open** from the Welcome screen and select the `defold/editor` directory. Choose to trust the project when prompted.
+
+2. Open the **Project** sidebar on the left of the IDE window and locate the `project.clj` file at root level. Right-click it and select **Add as Leiningen Project** from the context menu. The "Stub Generation Required" notification that might appear can be safely ignored.
```

Comment:
This is a crucial new step. Without it, the project just behaves like a folder of random source files. Nothing works.

I deleted `modules.xml` from the project and added it to `.gitignore` so that user will not get a name conflict for the project name. `modules.xml` works in conjunction with the `iml` file that is created from the Leiningen project, but we don't want to commit the `iml` file since it is managed by Cursive.